store/store.go

```diff
@@ -1119,6 +1128,19 @@ func findRev(needle snap.Revision, haystack []snap.Revision) bool {
 // The file is saved in temporary storage, and should be removed
 // after use to prevent the disk from running out of space.
 func (s *Store) Download(name string, downloadInfo *snap.DownloadInfo, pbar progress.Meter, user *auth.UserState) (path string, err error) {
+
+	if useDeltas() {
+		logger.Debugf("Available deltas returned by store: %v", downloadInfo.Deltas)
+	}
+	if useDeltas() && len(downloadInfo.Deltas) == 1 {
+		snapPath, err := s.downloadAndApplyDelta(name, downloadInfo, pbar, user)
+		if err == nil {
```

Comment:
(nitpick^2) You can put this into the same line: 
```
if snapPath, err := s.downloadAndApplyDelta(name, downloadInfo, pbar, user); err == nil { 
    return snapPath, nil
}
```
this is actually idiomatic go (even though it looks a bit unfamiliar coming from other languages).